src/bartholomew.rs

```diff
@@ -50,6 +50,16 @@ pub fn render(req: Request) -> Result<Response> {
         config.base_url = Some(url);
     }
     eprintln!("Base URL: {:?}", &config.base_url);
+    eprintln!("Prepend route info : {:?}", &config.prepend_route_info);
+
+    if let Some(true) = config.prepend_route_info {
```

Comment:
Is `if let <constant> = blah` preferred style to `if blah == <constant>`?  It reads oddly to me.

Perhaps clearer still would be `if config.prepend_route_info.unwrap_or(false)`?  (or `unwrap_or_default` according to taste)